### ISSUE TITLE
Updating to `stylelint@14.8.4` produces `Unexpected duplicate selector ""` `no-duplicate-selectors` rule

### ISSUE THREAD

**Issue by KeithGillette** [stylelint/stylelint#6112]
### What steps are needed to reproduce the bug?

The following selectors produce a `Unexpected duplicate selector ""` error:

```scss
%tt-btn-body {
	align-items: center;
	cursor: pointer;
	display: flex;
	justify-content: center;
	position: relative;

	i {
		font-size: 0.8rem;
		left: 0.5rem;
		margin-top: -0.5rem;
		position: absolute;
		top: 50%;
	}
}

%tt-btn-icon-segment-size {
	height: $tt-button-icon-only-size;
	padding: 0;
	width: $tt-button-icon-only-size;
}
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard-scss"
  ]
}
```

### How did you run Stylelint?

CLI

### Which version of Stylelint are you using?

14.8.4

### What did you expect to happen?

Passed linting, as with `stylelint@14.8.3` 

### What actually happened?

```
21:1  ✖  Unexpected duplicate selector "", first used at line 5  no-duplicate-selectors
```

**Comment by ybiquitous** [comment]
@KeithGillette Thanks for opening the issue and using the template.

This issue should be resolved by PR #6111. Please reopen this if you have further information. 

Duplicate of #6110

**Comment by KeithGillette** [comment]
Apologies for the duplicate issue, @ybiquitous. Since 6110 was already closed but the fix not yet released, I didn't see it.

**Comment by ybiquitous** [comment]
@KeithGillette Never mind. Thanks for taking the time.

**Comment by ybiquitous** [comment]
[14.8.5](https://github.com/stylelint/stylelint/releases/tag/14.8.5) released! 🎉
